frontend/src/layout/navigation-3000/sidepanel/SidePanel.scss

```diff
@@ -36,15 +38,31 @@
     .SidePanel3000__bar {
         display: flex;
         flex-direction: column;
-        justify-content: space-between;
-        width: 3rem;
+        align-items: center;
+        width: var(--side-panel-bar-width);
         height: 100vh;
-        overflow-y: auto;
+        overflow: hidden;
         user-select: none;
         border-left-width: 1px;
 
-        .LemonButton {
-            min-height: 2.25rem !important; // Reduce minimum height
+        .SidePanel3000__bar__tabs {
+            flex: 1;
+            width: var(--side-panel-bar-width);
+            overflow: hidden auto;
+
+            &::-webkit-scrollbar {
+                display: none;
+            }
+
+            .SidePanel3000__bar__tabs__rotate {
```

Comment:
These classnames go against [BEM](https://getbem.com/) concept – `SidePanel3000__bar__tabs__rotate` isn't Block Element, it's Block Element Element Element. So we still end up with a deeply nested structure AND monstrous class names.

`SidePanel3000__bar__tabs` should be just something like `SidePanel3000__tabs`, and `SidePanel3000__bar__tabs__rotate` - `SidePanel3000__tabs-content`. Both should also be at the top level of the file, there's no point nesting them (uncontrollable nesting is THE thing that BEM solves).

Reply:
Hmmmm I guess so. I like the alternative interpretation of BEM but sure happy to change it